module/VuFindSearch/src/VuFindSearch/Backend/Solr/Backend.php

```diff
@@ -94,6 +101,20 @@ public function __construct(Connector $connector)
         $this->identifier   = null;
     }
 
+    /**
+     * Set the limit for batch queries
+     *
+     * @param int $pageSize Records per Query
+     *
+     * @return void
+     */
+    public function setPageSize($pageSize)
+    {
+        if (!empty($pageSize)) {
```

Comment:
If you move the validation to the factory, you may not need this... but if you do want to validate, perhaps a better check would be whether the value is > 0, to avoid weird behavior caused by invalid configurations.

Reply:
Thank you Demian
I'm a little bit familiar with the test suite, but in this case I have no idea which test to add. I've given you write access to our repository to show me.